services/catalog/src/simcore_service_catalog/models/schemas/constants.py

```diff
@@ -0,0 +1,18 @@
+from typing import Final
+
```

Comment:
local ``_constants.py``?

Reply:
hmm for this... I'm not fond of this. I don't quite understand the idea. would that mean that these constants are only used withing schema/ folder?